deploy/kv/manual/cron.md

```diff
@@ -4,7 +4,13 @@ oldUrl:
   - /kv/manual/cron/
 ---
 
-<deno-admonition></deno-admonition>
+:::info Legacy Documentation
```

Comment:
This is not in the classic section, so should this disclaimer be here?

Reply:
I believe so because it references deploy classic. 
The KV docs definitely need some love. Not sure if should be part of this PR or an iteration afterwards